src/tokenize.cpp

```diff
@@ -22,6 +22,7 @@
 #include "language_tools.h"
 #include "punctuators.h"
 
+#define LE_COUNT(x)    cpd.le_counts[static_cast<size_t>(LE_ ## x)]
```

Comment:
Strongly typed enumerations don't implicitly cast to numbers, so all uses of `le_counts[LE_X]` need the cast. The macro simplifies that (and keeps line lengths under control).

Reply:
Then why not cast to int? We should rarely use this type of anything else other than a size.

Reply:
> We should rarely use this type of anything else other than a size.

Not even for array indices?

(Because `le_counts` is a C array, I think it does not matter, but if it was a `vector`, indexing with `int` would trip a `-Wconversion` warning. At any rate, negative vector indices make no sense. That said, `unsigned` would work just as well, if you'd prefer, but I used `size_t` mainly because that is the index type of `vector`.)